app/main/posts/modify/post-media.directive.js

```diff
@@ -25,7 +25,7 @@ function (
         template: require('./media.html'),
         link: function ($scope, element, attr, ngModel) {
             // Initialize media object
-            $scope.media = { id: null, file: null, caption: '', dataURI: null, changed: false };
+            $scope.media = { id: null, file: null, caption: '', dataURI: null, changed: false};
```

Comment:
should this have a `deleted: false` as well?

Reply:
Yep, true, they should all be there for clarity. Added it now :)